packages/www/components/Dashboard/WebhooksTable/index.tsx

```diff
@@ -118,6 +126,28 @@ const WebhooksTable = ({ title = "Webhooks" }: { title?: string }) => {
                 cursor: "pointer",
               },
             },
+            status: {
+              children: (
+                <Box>
+                  {!webhook.status ? (
+                    "-"
+                  ) : (
+                    <StatusBadge
+                      variant={
+                        webhook.status.lastFailure
+                          ? webhook.status.lastFailure.timestamp >=
+                            webhook.status.lastTriggeredAt
```

Comment:
I think we need a more "stable" logic here like for example showing the webhook as unhealthy, or at least a warning, if there has been any failure on the past X hours. Receiving a success does not mean that it is healthy, as we chatted already.

Reply:
The thing is, when we have a webhook that failed at lease once in the past X hours, we can display it unhealthy with a warning of the failure, but should we display both the last failure timestamp and the last trigger? Because a webhook with a failure in the last X hour would stay unhealthy until enough time as passed, without any clear indication that some hooks are successful. I have pushed a commit with this for clarity on what I mean!

Reply:
We can display only the most relevant information for the webhooks on the table, and if you want more details you go into the detail page and can see each value individually.

In the case of the unhealthy state, I think it's more relevant when was the last failure than when was the last trigger, so that's what I would display here.

Reply:
I like it, I added the Idle badge and displayed the detailed infos in the detail page